beater/config.go

```diff
@@ -81,25 +87,27 @@ func (c *FrontendConfig) SmapAccessor() utility.SmapAccessor {
 	return nil
 }
 
-var defaultConfig = Config{
-	Host:               "localhost:8200",
-	MaxUnzippedSize:    50 * 1024 * 1024, // 50mb
-	MaxHeaderBytes:     1048576,          // 1mb
-	ConcurrentRequests: 20,
-	ReadTimeout:        2 * time.Second,
-	WriteTimeout:       2 * time.Second,
-	ShutdownTimeout:    5 * time.Second,
-	SecretToken:        "",
-	Frontend: &FrontendConfig{
-		Enabled:      new(bool),
-		RateLimit:    10,
-		AllowOrigins: []string{"*"},
-		Sourcemapping: &Sourcemapping{
-			Cache: &Cache{
-				Expiration:      300,
-				CleanupInterval: 600,
+func defaultConfig() Config {
```

Comment:
While we're at it, it seems strange to copy `Config`s around rather than passing a pointer to _the_ config around.  What do you think about this returning a pointer?

Reply:
good point, changing to pass around *the* config everywhere.

Reply:
well played!